lib/mix/test/mix/tasks/xref_test.exs

```diff
@@ -1105,6 +1161,20 @@ defmodule Mix.Tasks.XrefTest do
       """)
     end
 
+    test "group with directly dependent files and cycle (json)" do
```

Comment:
IMO we don't need this test. The group works irrespective of the format.

Reply:
ack